### PR TITLE
2nd bugfix for #361 - forgot about register module

### PR DESCRIPTION
**Pull request by chris18890**
2nd bugfix for #361 - forgot about register module

### CODE REVIEW

**Comment by chris18890** [comment]
@ramgarden @elplatt another one to check :)

File: crm/modules/member/form.inc.php [hunk 1]
```diff
@@ -27,12 +27,6 @@
 */
 function member_add_form () {
     
-    // Ensure user is allowed to add members
-    if (!user_access('member_add')) {
```
Comment (ramgarden) [inline crm/modules/member/form.inc.php]: Why was this removed?

File: crm/modules/register/register.inc.php [hunk 2]
```diff
@@ -62,49 +62,6 @@ function register_form () {
...
-            )
-            , array(
-                'type' => 'text'
-                , 'label' => 'Emergency Phone'
-                , 'name' => 'emergencyPhone'
-            )
-        )
-    );
-    $form['fields'][] = array(
-        'type' => 'fieldset',
```
Comment (ramgarden) [inline crm/modules/register/register.inc.php]: You also removed all this membership info?

File: crm/modules/member/form.inc.php (hunk 1 above)
Comment (chris18890) [reply crm/modules/member/form.inc.php]: this is to allow the form to be rendered without anyone being logged in, so it can be used as part of the register form

**Comment by chris18890** [comment]
@ramgarden @elplatt replied to the comments, are you happy for this to be merged?

**Comment by ramgarden** [comment]
Yep. Satisfied my curiosity.  Looks OK to me.
